msgraph-developer-proxy/ProxyHost.cs

```diff
@@ -32,7 +69,8 @@ internal class ProxyHost {
         var command = new RootCommand {
             _portOption,
             _logLevelOption,
-            _recordOption
+            _recordOption,
+            _configFileOption!
```

Comment:
Added the option to the root command as well, so that it shows up in help

Reply:
Let's add a comment to explain that `_configFIleOption` gets set during the call to load `ProxyCommandHandler.Configuration` in the main program before the call to create the root command happens.
There's a few levels of indirection here and this could easily get missed during a refactor of the startup process

Reply:
Great point. Definitely makes sense to take note of that assumption here.